dev/default.env

```diff
@@ -22,9 +22,6 @@ COMPOSE_PROJECT_NAME=
 # MESSAGING_IMAGE_TAG=override-tag-name
 # MESSAGINGSERVICE_IMAGE_TAG=override-tag-name
 
-# Uncomment for deploys with traefik-managed ingress
-# COMPOSE_FILE=docker-compose.yaml:docker-compose.ingress.yaml
-
```

Comment:
would we not still want this option (and the `dev/docker-compose.ingress.yaml` file) for development?  obviously not on prod.

Reply:
Directly exposing Hapi to the internet is still an option, but it has its own override file now (added in #6):

[`dev/docker-compose.dev.fhir.yaml`](https://github.com/uwcirg/isacc-environments/blob/main/dev/docker-compose.dev.fhir.yaml)